### DISCUSSION TITLE
Need some help on ginvertermap and SDM630

### DISCUSSION THREAD

**Discussion by f0m3** [egguy/grott-home-assistant-add-on#34] [General]
I have on 3 Phase inverter with an SD630 and a second one behind the SDM630 only one phase.

Why does my config:
```
gmode: proxy
ginvtype: tl3
ginvtypemap: {'serial1': 'default','serial2': 'tl3'}
```
produce this error:
... 'mqttuser', 'ginvtypemap': {'[object Object]': None}}

**Comment by egguy** [comment]
Hello,

Did you put `{"NWO00415E3": "sph"}` or `{'NWO00415E3': 'sph'}` in the ginvertermap?

The second one is likely to cause trouble.

**Reply by f0m3** [reply]
i now added this into the gui textfield for ginvtypemap: `{"FVJ7CXXXX": "default", "SPE6A290XXX": "tl3"}`
When i switch to yaml mode this is in the yaml:
`ginvtypemap: "{\"FVJ7CXXXX\": \"default\", \"SPE6A290XXX\": \"tl3\"}"
`
I have values for both inverters but the SDM630 values dont show up.

Addition: There still also ist a `ginvtype: default`
When i try to remove that field its automatically added again.
